### ISSUE TITLE
AAD account ?

### ISSUE THREAD

**Issue by cara-lgrow** [citizenlabsgr/adopt-a-drain#214]
If you sign up for an AAD account (enter your email and password), do you get a welcome email? Right now that would be the same email you would get if you sign up for a drain....or do you only get a welcome email after you adopt your first drain? Trying to sort out which emails need to be updated/created.

**Comment by beyondtheteal** [comment]
Off the top of my head, there is a "thanks" email for the first and second adoption. I don't believe there is a welcome email.

**Comment by cara-lgrow** [comment]
but there would be an email (maybe not a 'welcome' email) if we set up some sort of password verification/check for a legitimate email address...

**Comment by beyondtheteal** [comment]
Correct, there will be a "Please verify you own this email account" email - issue #164. I feel like it'd be appropriate to also say "Welcome" in that email. I can supply rough draft verbiage for #164 if you'd like to work on the Welcome part.

**Comment by cara-lgrow** [comment]
Sounds  good!

**Comment by Wilfongjt** [comment]
No welcome email
1st adoption sends email
2nd adoption sends email
3rd email NO
